third_party/binary/BUILD

```diff
@@ -1,16 +1,15 @@
-GO_CI_LINT_VERSION = "1.54.2"
+GO_CI_LINT_VERSION = "1.58.2"
```

Comment:
Oh I'd forgotten about this. I dunno how much we use it (the github action doesn't)

Reply:
I use it to debug lint issues.